src/components/toolbar/legg-til-arbeidsliste-knapp.tsx

```diff
@@ -35,7 +36,7 @@ function ArbeidslisteKnapp() {
         (!arbeidslisteValgt && valgteBrukere.some(bruker => bruker.arbeidsliste.arbeidslisteAktiv)) ||
         arbeidslisteValgt;
 
-    if (skalSkjules) {
+    if (!skalVises && !erMock()) {
```

Comment:
Trenger man sjekk på mock her? (bare tenker at det kan bli litt rot dersom man sprer mock-sjekker utover i appen da det kan skape forvirring i utvikling eller feilsjekking lokalt senere.. om det gir mening :P )

Reply:
Tidlegare hadde vi logikk på linje 28-ish som sa "om vi manglar ident, vis komponenten". Vi mistenkte at den delen av logikken var for å få det til å fungere i lokalt. Eg har nettopp testa i dev og vel, vi hadde feil. Eg endrar logikken tilbake til slik den var før.